frontend/public/components/utils/datetime.ts

```diff
@@ -12,41 +12,41 @@ export const fromNow = (dateTime, now=undefined) => {
   const daysAgo = hoursAgo / 24;
 
   if (daysAgo > 548) {
-    return `${Math.round(daysAgo / 365)} years ago`;
+    return (options.omitSuffix) ? `${Math.round(daysAgo / 365)} years`:`${Math.round(daysAgo / 365)} years ago`;
   }
   if (daysAgo > 320) {
-    return 'a year ago';
+    return (options.omitSuffix) ? 'year':'a year ago';
   }
   if (daysAgo > 45) {
-    return `${Math.round(daysAgo / 30)} months ago`;
+    return (options.omitSuffix) ? `${Math.round(daysAgo / 30)} months`:`${Math.round(daysAgo / 30)} months ago`;
   }
   if (daysAgo > 26) {
-    return 'a month ago';
+    return (options.omitSuffix) ? 'month':'a month ago';
   }
   if (hoursAgo > 36) {
-    return `${Math.round(daysAgo)} days ago`;
+    return (options.omitSuffix) ? `${Math.round(daysAgo)} days`:`${Math.round(daysAgo)} days ago`;
   }
   if (hoursAgo > 22) {
-    return 'a day ago';
+    return (options.omitSuffix) ? 'day':'a day ago';
   }
   if (minutesAgo > 90) {
-    return `${Math.round(hoursAgo)} hours ago`;
+    return (options.omitSuffix) ? `${Math.round(hoursAgo)} hours`:`${Math.round(hoursAgo)} hours ago`;
   }
   if (minutesAgo > 45) {
-    return 'an hour ago';
+    return (options.omitSuffix) ? 'hour':'an hour ago';
   }
   if (secondsAgo > 90) {
-    return `${Math.round(minutesAgo)} minutes ago`;
+    return (options.omitSuffix) ? `${Math.round(minutesAgo)} minutes`:`${Math.round(minutesAgo)} minutes ago`;
   }
   if (secondsAgo > 45) {
-    return 'a minute ago';
+    return (options.omitSuffix) ? 'minute':'a minute ago';
```

Comment:
We might eventually want an option to include or omit the article `a` depending on context. For instance,

> Build my-build ran for minute.

vs

> Build my-build ran for a minute.

I think we can add that later as needed, though.